barcodecop/barcodecop.py

```diff
@@ -172,6 +172,9 @@ def main(arguments=None):
     match_options.add_argument(
         '-c', '--show-counts', action='store_true', default=False,
         help='tabulate barcode counts and exit')
+    match_options.add_argument(
+        '-t', '--tsv-counts', type=argparse.FileType('w'),
+        help='tabulate barcode counts and store as a TSV')
```

Comment:
Is there a specific reason to prefer TSV over CSV? I used tabs in "--show-counts" for display, but tend to default to csv for output.

Reply:
I tried both. I thought the resultant files were more readable with TSV, and no less parse-able by subsequent software. (ie. It's impossible for a barcode or count would have a \t in it). I'm glad to switch it to CSV (being a trivial change).

Reply:
csv would be better (and I am usually careful to use the csv module, even though it's unlikely for a delimiter to appear in this data). Also suggest adding metavar='FILE'